<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiceBookMarkWidget</class>
 <widget class="QDialog" name="DiceBookMarkWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiceShortCut</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::CopyAction</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="m_addBtn">
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../rolisteam.qrc">
           <normaloff>:/resources/icons/add_round.png</normaloff>:/resources/icons/add_round.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="m_removeBtn">
         <property name="text">
          <string notr="true">...</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../rolisteam.qrc">
           <normaloff>:/resources/icons/delete.png</normaloff>:/resources/icons/delete.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../rolisteam.qrc"/>
  <include location="../../../../rolisteam.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiceBookMarkWidget</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>603</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiceBookMarkWidget</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>598</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
